specs/gcpresource.spec

```
@@ -0,0 +1,82 @@
+# Model
+model:
+  rest_name: gcpresource
+  resource_name: gcpresources
+  entity_name: GCPResource
+  package: pandemona
+  group: pcn/infrastructure
+  description: Represents a GCP cloud resource such as a virtual machine.
+  extends:
+  - '@identifiable-stored'
+  - '@migratable'
+  - '@namespaced'
+  - '@zoned'
+
+# Indexes
+indexes:
+- - namespace
+  - selflink
+- - namespace
+  - numericID
+
+# Attributes
+attributes:
+  v1:
+  - name: data
+    description: The JSON-encoded data that represents the resource.
+    type: external
+    exposed: true
+    subtype: '[]byte'
+    stored: true
+    required: true
+    example_value: |-
+      {
+        "id": "12345",
+        "kind": "compute#instance"
+        ...
+      }
+
+  - name: kind
+    description: The specific kind of the resource.
+    type: enum
+    exposed: true
+    stored: true
+    required: true
+    allowed_choices:
+    - ComputeInstance
+    - ComputeSubnetwork
+    - ComputeNetwork
+    example_value: ComputeInstance
+
+  - name: numericID
+    description: A numeric resource ID that will mainly be used in RQL queries.
+    type: string
+    exposed: true
+    stored: true
+    required: true
+    example_value: 1234567
```

let's use a proper-length string :p 
```suggestion
    example_value: "0000000000000000000"
```